circle.yml

```diff
@@ -1,6 +1,6 @@
 checkout:
   post:
-    - ./ci_support/checkout_merge_commit.sh
+    - ./ci_support/fast_finish_ci_pr_build.sh
```

Comment:
Weird that we are losing `checkout_merge_commit.sh`. Could you please try re-rendering again?